### DISCUSSION TITLE
Livewire (Volt) Integration

### DISCUSSION THREAD

**Discussion by DevHoracioRodriguez** [laravel/nova-issues#5373] [Q&A] [answered]
Is there any plans to integrate Laravel [Volt](https://t.co/9uM05vSBLX)? Probably the question is too soon, but the issue that I have is that Nova uses Vue and I'm looking forward to work with livewire. 

If the above can't be answered, Is there a way to integrate livewire?

**Comment by crynobone** [comment, marked as answer]
Nova doesn't utilize Blade except for a single `layout.blade.php` and therefore will not be integrating with Laravel Volt.

**Reply by ziming** [reply]
is it possible to provide a iframe card or tool which we can then use to load the livewire component?
